inputs/templates/test/GatherBatchEvidence/GatherBatchEvidence.baf_from_vcf.json.tmpl

```diff
@@ -50,7 +50,7 @@
   "GatherBatchEvidence.gcnv_log_emission_sampling_median_rel_error": "0.001",
   "GatherBatchEvidence.gcnv_qs_cutoff": "30",
 
```

Comment:
Somehow I missed this comment - generating the VCFs now

Reply:
Actually I am going to punt on this since the RP bucket bug is currently preventing me from running GatherSampleEvidence. This workflow will not be needed much longer, once #18 is implemented.